docs/website/docs/user-guides/quickstart/quickstart.md

```diff
@@ -13,3 +13,12 @@ You have the option of choosing from the following frameworks for the quickstart
 * [Go](go)
 
 A full list of example applications can be viewed with the `odo registry` command.
+<details>
+<summary>Example</summary>
+
+```shell
+$ odo registry
+? Help odo improve by allowing it to collect usage data. Read about our privacy statement: https://developers.redhat.com/article/tool-data-c? Help odo improve by allowing it to collect usage data. Read about our privacy statement: https://developers.redhat.com/article/tool-data-collection. You can change your preference later by changing the ConsentTelemetry preference. Yes
+ NAME                          REGISTRY                DESCRIPTION
```

Comment:
This looks strange the output of the command should not be empty.

It should look something like this
```
$ odo registry
 NAME                          REGISTRY                DESCRIPTION
 dotnet50                      DefaultDevfileRegistry  Stack with .NET 5.0
 dotnet60                      DefaultDevfileRegistry  Stack with .NET 6.0
 dotnetcore31                  DefaultDevfileRegistry  Stack with .NET Core 3.1
 go                            DefaultDevfileRegistry  Go is an open source programming languag...
 java-maven                    DefaultDevfileRegistry  Upstream Maven and OpenJDK 11
 java-openliberty              DefaultDevfileRegistry  Java application Maven-built stack using...
 java-openliberty-gradle       DefaultDevfileRegistry  Java application Gradle-built stack usin...
 java-quarkus                  DefaultDevfileRegistry  Quarkus with Java
 java-springboot               DefaultDevfileRegistry  Spring Boot® using Java
 java-vertx                    DefaultDevfileRegistry  Upstream Vert.x using Java
 java-websphereliberty         DefaultDevfileRegistry  Java application Maven-built stack using...
 java-websphereliberty-gradle  DefaultDevfileRegistry  Java application Gradle-built stack usin...
 java-wildfly                  DefaultDevfileRegistry  Upstream WildFly
 java-wildfly-bootable-jar     DefaultDevfileRegistry  Java stack with WildFly in bootable Jar ...
 nodejs                        DefaultDevfileRegistry  Stack with Node.js 16
 nodejs-angular                DefaultDevfileRegistry  Angular is a development platform, built...
 nodejs-nextjs                 DefaultDevfileRegistry  Next.js gives you the best developer exp...
 nodejs-nuxtjs                 DefaultDevfileRegistry  Nuxt is the backbone of your Vue.js proj...
 nodejs-react                  DefaultDevfileRegistry  React is a free and open-source front-en...
 nodejs-svelte                 DefaultDevfileRegistry  Svelte is a radical new approach to buil...
 nodejs-vue                    DefaultDevfileRegistry  Vue is a JavaScript framework for buildi...
 php-laravel                   DefaultDevfileRegistry  Laravel is an open-source PHP framework,...
 python                        DefaultDevfileRegistry  Python is an interpreted, object-oriente...
 python-django                 DefaultDevfileRegistry  Django is a high-level Python web framew...
```

Reply:
Yeah true. Actuallly I was not sure whether it is same for everyone or not but now I got the idea. I am updating it. Thanks,

Reply:
added